src/settings.js

```diff
@@ -12,4 +12,37 @@ const TARGET_DATE = {
     seconds: 1
 }
 
+const WEEK_DAYS = {
+    1: 'Monday',
+    2: 'Tuesday',
+    3: 'Wednesday',
+    4: 'Thursday',
+    5: 'Friday',
+    6: 'Saturday',
+    7: 'Sunday'
+}
+
+const DAY_HOURS = {
+    1: '00:00',
+    2: '00:30',
+    3: '01:00',
+    4: '01:30',
+    5: '02:00',
+    6: '02:30',
+    7: '03:00',
+    8: '03:30',
+    9: '04:00',
+    10: '04:30',
+    11: '05:00',
+    12: '05:30',
+    13: '06:00',
+    14: '06:30',
+    15: '07:00',
+    16: '07:30',
+    17: '08:00',
+    18: '08:30',
+    19: '09:30',
+    20: '10:00'
+}
+
```

Comment:
redundant?

Reply:
yes redunant. Will use Dayjs weekdays and hours list.